### PR TITLE
W 10864152 mq update Tutorial and Queues screenshots hn

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by danieldatri — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/_partials/mq-msg-browser-details.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// tag::mqMsgBrowserDetails[]
+[cols="5a,10a,50a"]
+|===
+| (*1*) || Selected message
+| (*2*) || Message payload in the details pane
+| (*3*) |*Return Messages* icon | Return all messages to the queue (for example, to enable other applications to read the messages).
+
+Returning messages to the queue is a not-acknowledgment operation (NACK). The messages aren't altered. However, the TTL value you set when you created the queue determines how long the message is available before Anypoint MQ deletes it.
+
+If you switch from the *Message Browser* page, messages automatically return to the queue.
```
Comment (kbostrom) [inline modules/ROOT/pages/_partials/mq-msg-browser-details.adoc]: Pickyism (you know me...): when I read "switch," I expect to know what/where is being switched for what/where...which is not revealed here. So maybe...:
```suggestion
If you leave the *Message Browser* page, messages automatically return to the queue.
```
And maybe add something after "page," such as "...page before <this happens>, messages automatically...."

File: modules/ROOT/pages/_partials/mq-msg-browser-details.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// MQ Message Browser Details Table
+// tag::mqMsgBrowserDetails[]
+[cols="5a,10a,50a"]
+|===
+| (*1*) || Selected message
+| (*2*) || Message payload in the details pane
+| (*3*) |*Return Messages* icon | Return all messages to the queue (for example, to enable other applications to read the messages).
```
Comment (kbostrom) [inline modules/ROOT/pages/_partials/mq-msg-browser-details.adoc]: I think that, as with any list of items, if one item/column entry requires a period, then we should end all peer list items/entries with periods:
Selected message.
Message payload in the details pane.
Return all messages...messages).

Also, should "return" ^ be "returns"?

Comment (kbostrom) [reply modules/ROOT/pages/_partials/mq-msg-browser-details.adoc]: ^^^GLOBAL: is an actor performing the action (return) or is this just what the UI control does (returns)?

File: modules/ROOT/pages/mq-queues.adoc [hunk 3]
```diff
@@ -51,16 +50,13 @@ To create a standard queue or FIFO queue:
...
-+
+image::mq-blue-create.png["Blue plus icon at the top of the Destinations page"]
 . Select *Queue* or *FIFO Queue*.
 +
 For *Exchange*, see xref:mq-exchanges.adoc[Configure Message Exchanges].
 . Complete the fields in *Create Queue* or *Create FIFO Queue*:
 +
-.The screenshot shows the *Create Queue* popup.
-image::mq-create-queues.png[Create Queue popup]
+image::mq-create-queues.png["Create Queue popup"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: Should popup have a common noun object, like "menu" or "dialog box"? "popup" just sounds terribly visual to me....

File: modules/ROOT/pages/mq-queues.adoc [hunk 4]
```diff
@@ -185,49 +169,35 @@ For a FIFO queue, you can retrieve up to 10 messages.
...
 . Click *Get Messages*.
 . Click the message row to view the message payload in the details pane:
 +
-.The screenshot shows (*1*) the message row, (*2*) the message payload in the details pane, (*3*) the *Return Messages* icon, (*4*) the *Delete* icon, (*5*) the *Full Screen* icon, (*6*) the close details pane icon (*X*), and (*7*) the user properties.
-image::mq-tutorial-msg-browser-payload-details.png[Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane (X) icons]
-+
-* Click *Full Screen* to display the message payload in a separate window.
-* Click *X* to close the details pane.
-* Click *Delete* to delete the selected message.
+image::mq-tutorial-msg-browser-payload-details.png["Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane (X) icons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: Alternative that I can parse a bit more easily:
```suggestion
image::mq-tutorial-msg-browser-payload-details.png["Selected message row, payload in the details pane, and icons representing Return Messages, Delete, Full Screen, and close details pane (X)"]
```

File: modules/ROOT/pages/mq-queues.adoc [hunk 5]
```diff
@@ -242,10 +212,9 @@ For *Delivery Delay* (standard queue only) and *Message Group ID* (FIFO queue on
 . Click *Message Browser*, and then click *Get Messages*.
 . Click the message row to view the message payload in the details pane:
 +
-.The screenshot shows the CSV message payload, (*1*) the *Full Screen* icon, and (*2*) the close detail pane icon (*X*).
-image::mq-msg-browser-payload-details-csv.png[CSV payload, Full Screen icon, and close details pane icon (*X*)]
+image::mq-msg-browser-payload-details-csv.png["CSV payload, Full Screen icon, and close details pane icon"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: We included "(X" following "close details pane" in Line 176: should we not have, or should we add it here?

File: modules/ROOT/pages/mq-queues.adoc [hunk 6]
```diff
@@ -279,10 +246,9 @@ For *Delivery Delay* (standard queue only) and *Message Group ID* (FIFO queue on
 . Click *Message Browser*, and then click *Get Messages*.
 . Click the message row to view the message payload in the details pane:
 +
-.The screenshot shows the JSON message payload, (*1*) the *Full Screen* icon, and (*2*) the close details pane icon (*X*).
-image::mq-msg-browser-payload-details-json.png[JSON payload, Full Screen icon, and close details pane icon (X)]
+image::mq-msg-browser-payload-details-json.png["JSON payload, Full Screen icon, and close details pane icon"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: ditto comment at Line 215

File: modules/ROOT/pages/mq-queues.adoc [hunk 7]
```diff
@@ -322,17 +285,15 @@ If a queue is no longer needed, you can delete it. Any in-flight messages in the
...
 
-. In *Anypoint Platform > MQ*, click *Destinations* in the left pane.
-. Click the queue type for the queue to display its details in the right pane.
+. Select *Anypoint Platform* > *MQ* > *Destinations*.
+. Click the queue type to display its details.
 . Click the *Delete* icon in the details pane:
 +
-.The arrow shows the *Delete* icon in the details pane for deleting a queue.
-image::mq-delete-queue-icon.png[Delete icon in the details pane for deleting a queue]
+image::mq-delete-queue-icon.png["Delete icon in the details pane for deleting a queue"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: Would this suffice?
```suggestion
image::mq-delete-queue-icon.png["Delete icon in the details pane"]
```

File: modules/ROOT/pages/mq-queues.adoc [hunk 8]
```diff
@@ -450,10 +410,9 @@ image::mq-fifo-msg-groups.png[Message Group ID field for FIFO queues in the Mess
 To see which message group a message belongs to:
 
 . <<Get a Message from a Queue,Get a message from the queue>>.
-. In the *Message Browser*, check the details pane for the message: 
+. In the *Message Browser* page, check the details pane for the message:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc:413]: "check" = "search"?

File: modules/ROOT/pages/mq-queues.adoc [hunk 9]
```diff
@@ -450,10 +410,9 @@ image::mq-fifo-msg-groups.png[Message Group ID field for FIFO queues in the Mess
 To see which message group a message belongs to:
 
 . <<Get a Message from a Queue,Get a message from the queue>>.
-. In the *Message Browser*, check the details pane for the message: 
+. In the *Message Browser* page, check the details pane for the message: 
 +
-.The arrow shows the *Message Group ID* value in the details pane for a message in the queue.
-image::mq-fifo-msg-group-id.png[Message Group ID value in the details pane for a message]
+image::mq-fifo-msg-group-id.png["Message Group ID value in the details pane for a message"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]: Alternative:
```suggestion
image::mq-fifo-msg-group-id.png["Message Group ID value in the message details pane"]
```

File: modules/ROOT/pages/mq-queues.adoc [hunk 10]
```diff
@@ -527,14 +486,13 @@ To assign a DLQ to a queue:
...
-.The screenshot shows the DLQ fields in the *Create Queue* page.
-image::mq-create-q-dlq.png[DLQ fields in the Create Queue page]
+image::mq-create-q-dlq.png["DLQ fields in the Create Queue page"]
 
 
 * *Dead Letter Queue Name* 
 +
 Choose a previously created queue name from the drop-down list.
-* *Reroute after 10 attempts* (Optional)
+* *Delivery attempts before reroute* (Optional)
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]:
```suggestion
* *Delivery attempts before reroute* 
```

File: modules/ROOT/pages/mq-queues.adoc [hunk 11]
```diff
@@ -527,14 +486,13 @@ To assign a DLQ to a queue:
...
 +
 Specify how many times Anypoint MQ attempts to deliver messages in the queue before rerouting the message to the dead-letter queue. If not specified, the default value is 10 tries. This value ranges from 1 to 1000 attempts.
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-queues.adoc]:
```suggestion
Optionally specify how many times Anypoint MQ attempts to deliver messages in the queue before rerouting the message to the dead-letter queue. If not specified, the default value is 10 tries. This value ranges from 1 to 1000 attempts.
```
I also like the style of beginning optional steps with the condition under which you might take the action: "If you want to specify that...fewer or more times than the default of 10 times, specify...."

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 12]
```diff
@@ -118,44 +111,35 @@ include::partial$mq-user-props.adoc[tag=mquserprops]
 ----
 . Click *Send*.
 +
-Next, you verify that the messages arrived in the queue.
+Next, verify that the messages arrived in the queue.
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc:114]:
```suggestion
Next, verify that the messages arrived in the queue:
```
This is introducing a new set of steps?

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 13]
```diff
@@ -118,44 +111,35 @@ include::partial$mq-user-props.adoc[tag=mquserprops]
...
+. Click the queue type for `myDemoQueue` to display its details.
 +
 The details pane shows the number of messages in the queue:
 +
-.The arrow shows the three messages that you sent to the `myDemoQueue` queue in the details pane.
-image::mq-tutorial-msgs-in-queue.png[Three messages in the myDemoQueue queue in the details pane]
-. In the *Destinations* page, click `myDemoQueue`.
-. In the settings page, click *Message Browser* in the left pane:
+image::mq-tutorial-msgs-in-queue.png["Three messages in the myDemoQueue queue in the details pane"]
+. In the details pane, click the queue details label, `myDemoQueue`.
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
. In the details pane, click the queue details label `myDemoQueue`.
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 14]
```diff
@@ -118,44 +111,35 @@ include::partial$mq-user-props.adoc[tag=mquserprops]
...
 +
-.The screenshot shows (*1*) the message row, (*2*) the message payload in the details pane, (*3*) the *Return Messages* icon, (*4*) the *Delete* icon, (*5*) the *Full Screen* icon, (*6*) the close details pane icon (*X*), and (*7*) the user properties.
-image::mq-tutorial-msg-browser-payload-details.png[Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane (X) icons]
+Anypoint MQ retrieves the messages from the queue and displays them in the Message Browser.
+. Click the message row to view the message payload in the details pane:
 +
-* Click *Full Screen* to display the message payload in a separate window.
-* Click *X* to close the details pane.
-* Click *Delete* to delete the selected message.
+image::mq-tutorial-msg-browser-payload-details.png["Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane icons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]: ditto edit to Line 176 in [mq-queues.adoc](https://github.com/mulesoft/docs-mq/pull/101/files#diff-d9e706978faf997d0694f226d6ab7c9a3770dd8476a80f7174bde0993dd47632)

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 15]
```diff
@@ -201,99 +189,81 @@ If enabled, the password program inserts passwords in fields in Design Center, w
...
+|===
+
+. Click the plus (+) icon to the right of the HTTP Listener trigger:
 +
-. In the *Select a component* window, enter `Mu` to search for the Mule Anypoint MQ Connector:
+image::mq-tutorial-dc-plus-icon.png["Plus icon"]
 +
-.The arrow shows the Mule Anypoint MQ Connector.
-image::mq-tutorial-dc-mq-search.png[Search for the Mule Anypoint MQ Connector]
+. In the *Select a component* window, enter `mule an` to search for the Mule Anypoint MQ Connector:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc:215]: Connector names when fully spelled out are typically "Anypoint Connector for XYZ (XYZ Connector)."

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 16]
```diff
@@ -201,99 +189,81 @@ If enabled, the password program inserts passwords in fields in Design Center, w
...
 +
-.. In the *MQ* page, click *Client Apps* and click `DemoClientApp` to display the details pane.
-.. In the *MQ* details pane, click the *Copy* button for the *Client App ID* field and paste the value in the *Client App ID* field in the *Design Center* page.
-.. In the *MQ* details pane, click the *Copy* button for the *Client Secret* field and paste the value in the *Client Secret* field in the *Design Center* page:
+image::mq-tutorial-url.png["Copy button in the Destinations page"]
+.. Click *Client Apps* and click `DemoClientApp` to display the details pane.
 +
-.The screenshot shows (*1*) the *Client App ID* field and (*2*) *Client Secret* field in the *Design Center* page.  
-image::mq-tutorial-dc-client-id-secret.png[Client App ID field and Client Secret field]
+image::mq-tutorial-client-apps-detail.png["Client app name, Copy, and Show buttons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
image::mq-tutorial-client-apps-detail.png["Client app name and Copy and Show buttons"]
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 17]
```diff
@@ -326,24 +295,23 @@ curl -X POST \
...
 . Specify the number of messages (1-500) and the time to poll (1-3600 seconds).
 . Click *Get Messages*.
-. Click the message row to view the message payload that you sent in the `curl` command in the details pane:
 +
-.The screenshot shows (*1*) the message row, (*2*) the message payload, (*3*) the *Return Messages* icon, (*4*) the *Delete* icon, (*5*) the *Full Screen* icon, and (*6*) the close details pane icon (*X*).
-image::mq-tutorial-msg-browser-payload-details-curl.png[Selected message row, payload, Return Messages icon, Delete icon, Full Screen icon, and close details pane (X) icons]
+Anypoint MQ retrieves the messages from the queue and displays them in the Message Browser.
+. Click the message row to view the message payload in the details pane:
++
+image::mq-tutorial-msg-browser-payload-details.png["Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane icons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]: ditto suggestion for Line 176, [mq-queues.adoc](https://github.com/mulesoft/docs-mq/pull/101/files#diff-d9e706978faf997d0694f226d6ab7c9a3770dd8476a80f7174bde0993dd47632)

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 18]
```diff
@@ -368,78 +336,74 @@ Ensure that the queue you created in *Anypoint Platform* > *MQ* is also in this
...
-.The arrow shows the *Exchange* icon in the Studio 7 toolbar.
-image::mq-tutorial-studio7-exchange.png[Exchange icon in the Studio 7 toolbar]
+image::mq-tutorial-studio7-exchange.png["Exchange icon in the Studio 7 toolbar"]
 . Enter your Anypoint Platform username and password.
-. In Exchange, select *Provided by MuleSoft*, and then search for the Anypoint MQ Connector:
+. In Exchange, click the *Provided by MuleSoft* navigation menu, select *Connectors* from the dropdown, and then search for Anypoint MQ Connector:
 +
-.The screenshot shows (*1*) the *Provided by MuleSoft* option and (*2*) the search field.
-image::mq-tutorial-exchange.png[Provided by MuleSoft option and search field in Exchange]
+image::mq-tutorial-exchange.png["Provided by MuleSoft menu, Connectors dropdown, and search field in Exchange"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]: Again, I wonder if we should supply a noun object for "dropdown" in the alttext.

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 19]
```diff
@@ -368,78 +336,74 @@ Ensure that the queue you created in *Anypoint Platform* > *MQ* is also in this
...
+image::mq-tutorial-studio7-palette.png["HTTP Connector and Listener operations in the Mule Palette"]
 +
-.The arrow shows the *Publish* operation in the Studio 7 canvas.
-image::mq-tutorial-studio7-flow.png[Publish operation in the Studio 7 canvas]
+[cols="5a,15a,50a"]
+|===
+| (*1*) | HTTP Connector| Click to display the Listener operation.
+| (*2*) | Listener operation| Drag to the Mule Palette.
+|===
+. Click *Anypoint MQ*, and then drag *Publish* to the *Process* area to the right of the Listener on the canvas:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
. Click *Anypoint MQ* and then drag *Publish* to the *Process* area to the right of the Listener on the canvas:
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 20]
```diff
@@ -368,78 +336,74 @@ Ensure that the queue you created in *Anypoint Platform* > *MQ* is also in this
...
 +
-.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.  
-image::mq-tutorial-client-apps-detail.png[Client App ID field, Show and Copy buttons]
+image::mq-tutorial-url.png["Copy button in the Destinations page"]
+.. Click *Client Apps* and click `DemoClientApp` to display the details pane.
 +
-.. In the *MQ* page, click *Client Apps* and click `DemoClientApp` to display the details pane.
-.. In the *MQ* details pane, click the *Copy* button for the *Client App ID* field and paste the value in the *Client App ID* field in the *Anypoint MQ Config* window.
-.. In the *MQ* details pane, click the *Copy* button for the *Client Secret* field and paste the value in the *Client Secret* field in *Anypoint MQ Config* window:
+image::mq-tutorial-client-apps-detail.png["Client app name, Copy, and Show buttons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
image::mq-tutorial-client-apps-detail.png["Client app name and Copy and Show buttons"]
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 21]
```diff
@@ -477,30 +440,27 @@ in the Studio 7 *MQ Publish* > *Destination* field matches the name of the queue
...
+image::mq-tutorial-get-messages.png["Message Browser navigation menu"]
 . Specify the number of messages (1-500) and the time to poll (1-3600 seconds).
 . Click *Get Messages*.
++
+Anypoint MQ retrieves the messages from the queue and displays them in the Message Browser.
 . Click the message row to view the message payload in the details pane:
 +
-.The screenshot shows (*1*) the message row, (*2*) the message payload, (*3*) *Return Messages* icon, (*4*) *Delete* icon, (*5*) *Full Screen* icon, and (*6*) close details pane icon (*X*).
-image::mq-tutorial-msg-browser-payload-details-curl-rtm.png[Selected message row, payload, Return Messages icon, Delete icon, Full Screen icon, and close details pane icon (X)]
+image::mq-tutorial-msg-browser-payload-details.png["Selected message row, payload in the details pane, and Return Messages, Delete, Full Screen, and close details pane icons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]: ditto edit to Line 176 [mq-queues.adoc](https://github.com/mulesoft/docs-mq/pull/101/files#diff-d9e706978faf997d0694f226d6ab7c9a3770dd8476a80f7174bde0993dd47632)

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 22]
```diff
@@ -568,39 +522,33 @@ To configure a Mule 3 app in Studio 6:
 . Install the Anypoint MQ Connector:
 .. Click the *Exchange* icon in the toolbar:
 +
-.The arrow shows the *Exchange* icon in the Studio 6 toolbar.
-image::mq-tutorial-studio6-exchange.png[Exchange icon in the Studio 6 toolbar]
+image::mq-tutorial-studio6-exchange.png["Exchange icon in the Studio 6 toolbar"]
 .. Enter your Anypoint Platform username and password.
-.. In Exchange, select *Provided by MuleSoft*, and then search for Anypoint MQ Connector:
+. In Exchange, click the *Provided by MuleSoft* navigation menu, *Connectors* from the dropdown, and then search for Anypoint MQ Connector:
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
. In Exchange, click the *Provided by MuleSoft* navigation menu, select *Connectors* from the dropdown, and then search for Anypoint MQ Connector:
```

File: modules/ROOT/pages/mq-tutorial.adoc [hunk 23]
```diff
@@ -613,47 +561,43 @@ The HTTP properties window now indicates that there are no errors.
...
 +
-.The arrow shows the *Copy* button on the Anypoint MQ *Destinations* page.
-image::mq-tutorial-url.png[Copy button on the Destinations page]
-. Copy the client app ID and client secret from the *Client Apps* page in Anypoint MQ to Studio:
+image::mq-tutorial-url.png["Copy button in the Destinations page"]
+.. Click *Client Apps* and click `DemoClientApp` to display the details pane.
 +
-.The screenshot shows (*1*) the *Client App ID* field, (*2*) *Show* button, and (*3*) *Copy* button.  
-image::mq-tutorial-client-apps-detail.png[Client App ID field, Show and Copy buttons]
+image::mq-tutorial-client-apps-detail.png["Client app name, Copy, and Show buttons"]
```
Comment (kbostrom) [inline modules/ROOT/pages/mq-tutorial.adoc]:
```suggestion
image::mq-tutorial-client-apps-detail.png["Client app name and Copy and Show buttons"]
```

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/mq-tutorial.adoc (hunk 12 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-tutorial.adoc:114]: No.

File: modules/ROOT/pages/mq-tutorial.adoc (hunk 15 above)
Comment (hannanelson) [reply modules/ROOT/pages/mq-tutorial.adoc:215]: The label of the connector in Exchange is "Mule Anypoint MQ Connector".

**Review by kbostrom — APPROVED** [review]
RA

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**